### PR TITLE
preScreenIntegrityReviewUpdatedWebhook

### PR DESCRIPTION
**Pull request by ivyc81**
Updated doc to include preScreenIntegrityReviewUpdatedWebhook

### CODE REVIEW

File: docs/webhooks/index.html [hunk 1]
```diff
@@ -625,6 +626,36 @@ <h4 id="companyTestSessionFinished">companyTestSessionFinished</h4>
...
+              <pre><code>{
+  event: 'preScreenIntegrityReviewUpdated',
+  triggeredOn: number,
+  payload: {
+    testSessionId: string,
+    testId: string,
+    testTitle: string,
+    candidateEmail: string,
+    candidateName: string,
+    integrityReviewSuggested: boolean | null,
```
Comment (Albinutte) [inline docs/webhooks/index.html:644]: Following the discussion in the main pull request we might want to update this type to be just `boolean`.

File: docs/images/candidate-events.png
Comment (Albinutte) [inline docs/images/candidate-events.png:1]: It appears that there's unnecessary white space to the left of the candidate, could you remove it?

File: docs/images/candidate-events.svg
Comment (Albinutte) [inline docs/images/candidate-events.svg:1]: Here, too, could you remove white space to the left of the candidate?

**Review by Albinutte — APPROVED** [review]
:shipit:

**Comment by ivyc81** [comment]
Waiting for https://github.com/CodeSignal/codesignal/pull/34525 to be on prod before merging.

**Pull request merged**